.gitignore

```diff
@@ -26,3 +26,6 @@ go.work
 
 # GolangCI-Lint cache
 /cache
+
+# Intellij
+.idea/
```

Comment:
I'd prefer not to have a setup-based entry in our `.gitignore`. You could add `.idea/` to your global `.gitignore`, so that it never gets added to the staging area.